crates/cairo-coverage/src/inner/config.rs

```diff
@@ -0,0 +1,72 @@
+use crate::inner::data_loader::deserialize;
+use crate::inner::data_loader::types::StatementMap;
+use anyhow::{Context, Result};
+use cairo_lang_sierra::program::{Program, VersionedProgram};
+use camino::Utf8PathBuf;
+use trace_data::CallTrace;
+
+#[allow(dead_code)] // Temporary
+struct CoverageConfig {
+    call_trace: CallTrace,
+    program: Program,
+    statement_map: StatementMap,
+}
+
+impl CoverageConfig {
+    #[allow(dead_code)] // Temporary
+    pub fn new(
+        call_trace_path: &Utf8PathBuf,
+        versioned_program_path: &Utf8PathBuf,
```

Comment:
Why is `versioned_program_path` a parameter here? Shouldn't we rather take this information from `call_trace` -> `cairo_execution_info` -> `source_sierra_path`?